### PR TITLE
Update design for Devise pages

### LINKED ISSUE #436: Update layout of Login/User profile page (Devise layout)
Related to #444.

The pages related to the user account tasks (registration, login, user profile) are unnecessarily narrow, creating a poor experience when using a browser with a narrow viewport:

Screen Shot 2022-02-28 at 3 01 39 PM

---

This appears to happen because the [Devise layout file](https://github.com/pod4lib/aggregator/blob/c4552721cf8b4e9532b36b5f359cf991cb8254e9/app/views/layouts/devise.html.erb) includes the CSS class `w-25`, which tells that div to use only 25% of the container width. 25% width is sufficient when the viewport is a decent desktop size (though even there wider than 25% would be perfectly fine), but not at narrower viewports.

I'd suggest replacing the `w-25` class with some Bootstrap column classes, which aren't currently used in this layout file. ~For example using `col-md-6` might work for this div, using half the page width at `md` and larger, while using the full width on mobile browsers. That approach might require adding a `row` class somewhere above, not sure off-hand.~ See #444 for a more thought-through suggestion.

If we want to use the simplest possible approach, I think just removing the `w-25` would be okay (form inputs would use the full width of the page, which is unnecessary at `md` and wider, but is much better than being too narrow), but doing #444 instead of this ticket would result in a much better overall user experience.

### LINKED ISSUE #444: Update layout and content of Devise-based pages
In issue #436 I suggested a simple fix to the Devise-based sign-up/login pages. That issue only addresses the current layout problem with those pages, though, and as I work through the app more I've noticed other issues with the sign-up/login pages that ideally would also be addressed to create a more attractive and clear user experience. If we don't have time to revamp these pages as suggested below, #436 is the minimum we should do, but it would be better to do the more thorough revamp suggested below.

## Basic layout changes
As I noted in #436, I think we should introduce Bootstrap columns into these pages for better layout. In the mockups below, I've used a wrapper div with `class="col-md-8 offset-md-2` to position the form in the middle of the page, at the `md` breakpoint and larger, and keeping the width of the forms plenty wide but not full-page width. At below `md` I think it will be fine to give the form full-page width.

Other consistent layout choices:
- Center the page title
- Add a horizontal rule/bottom border to separate the main form from the potentially relevant but secondary ("Sign up", "Log in") and tertiary ("Forgot your password?") link actions
- Left-align the secondary actions for a given page with the left-edge of the form
- Right-align the tertiary actions with the right-edge of the form

Note that some page titles, field labels, and button labels are the same as the Devise default but others are updated a bit for better clarity.

Here's how the various pages would more or less look after those changes are made:

## Set password
This is the form you get if you are emailed an invite to sign up with POD.

set-password

---

## Sign up

sign-up

---

## Log in

log-in

---

## Forgot password

forgot-password

---

## Resend confirmation instructions

resend

---

### PR DESCRIPTION
**Pull request by marlo-longley**
Fixes #444 and by extension fixes #436 @ggeisler 

Aside from HTML and CSS changes, this PR adds a new entry to `locales/en.yml` to override the default label for email addresses.

Notes:
- The button colors, which are tied to the bootstrap theme, I've left as default/primary, so for now they don't match Gary's mockups. But this should become aligned after addressing  #459

Screen Shot 2022-03-16 at 16 39 23
Screen Shot 2022-03-16 at 16 38 54
Screen Shot 2022-03-16 at 16 39 30

### CODE REVIEW

**Comment by ggeisler** [comment]
Looks good to me @marlo-longley (but will defer to a developer for formal review.

File: app/views/layouts/devise.html.erb
Comment (thatbudakguy) [inline app/views/layouts/devise.html.erb:5]: the fields look super long to me at this size. what if we did `col-md-6 offset-md-3`?

**Review by thatbudakguy — APPROVED** [review]
everything looks fine to me, except that the text fields seem super wide on desktop.

File: app/views/layouts/devise.html.erb
Comment (ggeisler) [reply app/views/layouts/devise.html.erb:5]: The original design included links both left- and right-aligned below the horizontal rule, and if I remember correct in some cases (e.g., the Forgot your password? page) going smaller than `col-md-8` put the adjacent links too close together to look good. And there's no harm in an overly wide input field on this page so that seemed the best compromise.

I guess if we're going to stick to left-aligned only links, the above situation is no longer in play, so sure, I guess we could go with `col-md-6 offset-md-3` as long as that still works at all breakpoints.

Comment (marlo-longley) [reply app/views/layouts/devise.html.erb:5]: Oops, when I commented my view hadn't updated since Gary's comment.

Comment (thatbudakguy) [reply app/views/layouts/devise.html.erb:5]: @ggeisler thx for context. i think let's just commit to keeping everything left-aligned so we can shrink the overall container a bit on large screens.

**Comment by marlo-longley** [comment]
@thatbudakguy @ggeisler here's the new look; pushed the commit.
Old
Screen Shot 2022-03-16 at 16 39 30
New
Screen Shot 2022-03-17 at 11 28 02

**Comment by ggeisler** [comment]
@marlo-longley Since this hasn't been merged yet, note that the decision made in #475 that there be no direct user registration means we should remove the "Sign up" link from all user account pages. 

Not sure if you'd prefer to do that via another commit to this PR or if you want to do it as a separate issue after this is merged.

**Comment by thatbudakguy** [comment]
@ggeisler I took a look at this and I think it'd be better to do it as a separate issue/PR, since I think we will also need to remove any language that appears when the user gets to the sign up page via a different flow:

accesses login-only page -> prompted to sign in or sign up -> clicks sign up

there are some other related things to remove on the backend too, like the reCaptcha and form for signing up (which have some existing problems as noted in #266).

**Comment by thatbudakguy** [comment]
new issue for restricting signup is #499.

**Pull request merged**
